### PR TITLE
Added tooltip for labels

### PR DESCRIPTION
**Pull request by rsanchez15**
Signed-off-by: RaulSanchez <raul@eprosima.com>

### CODE REVIEW

File: qml/InfoToolTip.qml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import QtQuick.Controls 2.15
+import Theme 1.0
+
+Item {
+    anchors.fill: parent
+    property string text: ""
+    ToolTip {
+        id: tooltip
+        text: parent.text
+        delay: 250
```
Comment (jparisu) [inline qml/InfoToolTip.qml:25]:
```suggestion
        delay: 0
```
If there is a delay it is a possibility that the user does not realize the help exists

**Review by jparisu — CHANGES_REQUESTED** [review]
1. Add documentation comment to this helps.

2. NIT: For tootip indentation, I would suggest to:
  1. Use a common line length equal to all of them
  2. Break line with points
As it is some boxes are difficult to read as there are too much text condensed and each sentence is not easily differencianted.

File: qml/InfoToolTip.qml (hunk 1 above)
Comment (rsanchez15) [reply qml/InfoToolTip.qml:25]: I'd keep the delay as this is how it works in all applications

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**